scripts/disk_check.py

```diff
@@ -40,6 +40,8 @@
 WORK_DIR = "/run/mount/work"
 MOUNTS_FILE = "/proc/mounts"
 
+FREE_SPACE_THRESHOLD = 1024
```

Comment:
>FREE_SPACE_THRESHOLD [](http://example.com/codeflow?start=0&length=20)

what is the unit? could you add code comment?

Reply:
Fixed, unit is file system block, on ext4 this means 1024*4096 = 4MB